docs/catalog/ai/energy-efficent-framework.md

```diff
@@ -18,6 +18,12 @@ Training an AI model implies a significant carbon footprint. The underlying fram
 
 Typically, AI/ML frameworks built on languages like C/C++ are more energy efficient than those built on other programming languages.
 
+Following provides an interesting view on a [normalized analysis regarding languages and their energy footprint(https://sites.google.com/view/energy-efficiency-languages/results?authuser=0)].
+
+Take into accoutn that in the Cloud for example the processing time of non CPU intensive applications is mostly spent idling because of the strong interservice dependencies that are network related (In other words a request through the network is typically 10000+ times slower than a CPU operation). 
+
+Libraries like TensorFlow or PyTorch allow to leverage GPUs easily that would have a different consumption scheme compared to CPUs, TPU for TensorFlow typically outperform as well CPUs: All these points need to be properly analyzed before selecting a language that might bring drawbacks in terms of staffing skilled people.
```

Comment:
```suggestion
Libraries like TensorFlow or PyTorch allow easy GPU utilization, which has a different consumption pattern compared to CPUs. TPUs (Tensor Processing Units) in TensorFlow typically outperform CPUs. All these points need proper analysis before selecting a language, as it may impact staffing skilled personnel.
```